blog/__init__.py

```
@@ -0,0 +1 @@
+from .models import Post
```

Ruff (0.8.2)

`.models.Post` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)